### PR TITLE
Make zksync default, add gnt-driver feature flag

### PR DESCRIPTION
**Pull request by maaktweluit**

### CODE REVIEW

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'golemfactory/yagna-integration'
+          branch: 'mwu/multi-payment'
```
Comment (maaktweluit) [inline .github/workflows/integration-test.yml:18]: TODO: Revert before merging, after merging `ya-int`

File: Cargo.toml [hunk 2]
```diff
@@ -10,7 +10,7 @@ license = "GPL-3.0"
 edition = "2018"
 
 [features]
-default = ['gnt-driver', 'gftp/bin']
+default = ['zksync-driver', 'gftp/bin']
```
Comment (Wiezzel) [inline Cargo.toml:13]: I'd say we should include both in the default build.

Comment (maaktweluit) [reply Cargo.toml]: Great idea! updated

**Review by Wiezzel — DISMISSED** [review]

File: .github/workflows/integration-test.yml [hunk 3]
```diff
@@ -15,6 +15,7 @@ jobs:
...
           token: ${{ secrets.YAGNA_WORKFLOW_TOKEN }}
 
       - name: Configure python 3.7
```
Comment (zakaprov) [inline .github/workflows/integration-test.yml:21]: This workflow is going to be removed in https://github.com/golemfactory/yagna/pull/723. If you'd like to get it running until then anyway, please also update the python version used here to 3.8.

Comment (maaktweluit) [reply .github/workflows/integration-test.yml:21]: Good one, will do when removing the test branch

**Review by zakaprov — APPROVED** [review]

**Review by zakaprov — APPROVED** [review]

**Review by Wiezzel — APPROVED** [review]

**Pull request merged**